src/probability/probability_mass_function/basic.lean

```diff
@@ -131,8 +155,7 @@ le_antisymm (p.to_outer_measure_mono (h.symm ▸ (set.inter_subset_left t p.supp
 @[simp]
 lemma to_outer_measure_apply_fintype [fintype α] :
   p.to_outer_measure s = ↑(∑ x, (s.indicator p x)) :=
```

Comment:
Why is there still a coercion here? Same question below in several lemmas like `to_measure_apply` and `to_measure_apply'` and `to_measure_apply_finset` and `to_measure_apply_of_finite` and `to_measure_apply_fintype`.

Reply:
Cleared out a few more implicit coercions as well as these explicit ones